src/scenes/ModulesRouter/ModuleEventSessionView.tsx

```diff
@@ -121,7 +136,7 @@ const ModuleEventView = () => {
 
               <Box dangerouslySetInnerHTML={{ __html: event?.description || '' }} />
 
-              {event?.additionalLinks?.length && (
+              {/* {event?.additionalLinks?.length && (
```

Comment:
comment-out intentional?

Reply:
Trying to figure out where this data is supposed to come from, will be updated once the back is sorted.